src/python/grpcio_tests/tests/BUILD.bazel

```diff
@@ -0,0 +1,8 @@
+py_library(
+    name = "bazel_patch",
+    srcs = ["bazel_patch.py"],
+    visibility = ["//visibility:public"],
```

Comment:
Please keep this visibility restricted to a whitelist of specific tests that need it.